__init__.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+
+PY37 = sys.version_info >= (3, 7)
+
+
+if not PY37:
+    from pep562 import Pep562
+
+    Pep562(__name__)
```

Comment:
@im-harrydoyle Can you clarify why you have this logic here if the same is present in each of the lower-down init files?

Reply:
@BenRKarl You're right, we don't need it in the top-level init.py. It's only needed where there is a def __dir__ or def __getattr__, which is in the version directories.

Reply:
Thank you - I thought so, I just wanted to make sure I wasn't missing something.

Reply:
@BenRKarl I've now tested the 8.1.0 release, having removed the content of the top-level init.py, and can confirm it works as expected Thanks for creating this version!

Reply:
I guess that infectious and im-harrydoyle are the same person? Either thanks for your contribution!

Reply:
@wihl yes, in that Infectious Media is the company and I (Harry) am responsible for the tech side of the business, so I'm managing our presence on github :-)

Reply:
We want to thank (immortalize?) you in the ChangeLog. Which alias do you prefer?

Reply:
Let's go for infectious. It was a team effort at our end :-)